manifests/config.pp

```diff
@@ -52,8 +58,20 @@
     'blank': {
       $changes = "clear ${name}"
     }
-    default: {
-      fail "Unknown value for ensure '${ensure}'"
+    default: {}
+  }
+
+  if !defined('$changes') {
```

Comment:
Why not put this block inside the `default` block above?

Reply:
Yeah. Sounds good, and works the same. Updating